app/src/main/res/values/strings.xml

```diff
@@ -10,5 +10,28 @@
     <string name="onboarding_parking_tutorial">If convenient, park at a bike rack. If not, park close to the edge of the sidewalk closest to the street. Do not block sidewalks, doors or ramps.</string>
     <string name="onboarding_rules_tutorial">You must be 18 years or and older with a valid driving licence to operate a scooter. Please follow all street signs, signals and markings, and obey local traffic laws.</string>
     <string name="onboarding_next_button">Next</string>
+    <string name="get_started_button">Get started</string>
```

Comment:
Separate your strings a bit into regions for each feature. It makes the file easier to read
Add an empty line a comment line <!-- region FeatureName --> between each section